.babelrc

```diff
@@ -1,14 +1,35 @@
 {
-  "presets": [
-    [
-      "next/babel",
-      {
-        "styled-jsx": {
-          "plugins": [
-            "styled-jsx-plugin-postcss"
-          ]
-        }
-      }
-    ]
-  ]
+  "env": {
+    "development": {
+      "presets": [
+        [
+          "next/babel",
+          {
+            "styled-jsx": {
+              "plugins": [
+                "styled-jsx-plugin-postcss"
+              ]
+            }
+          }
+        ]
+      ]
+    },
+    "test": {
```

Comment:
Isn't babel-jest missing in your dependencies? 🤔 https://github.com/facebook/jest/tree/master/packages/babel-jest

Reply:
not according to the example in next :/ but i'll try it tonight and report. https://github.com/zeit/next.js/blob/canary/examples/with-jest/.babelrc